app/views/layouts/application.html.erb

```diff
@@ -3,37 +3,57 @@
 
 <head>
   <title>GarfieldReview</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <%= stylesheet_link_tag 'application', media: 'all' %>
   <%= csrf_meta_tags %>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" href="../images/fav_icon.png" type="image/x-icon">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/overlayscrollbars/1.9.1/css/OverlayScrollbars.min.css'>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  <!-- Bulma Version 0.8.2-->
+
+  <style>
+    .columns:nth-child(2) {
+      margin-top: 10rem;
+    }
+  </style>
+  <link rel="stylesheet" href="../../../app/assets/stylesheets/customClasses.css">
```

Comment:
woh on multiple fronts
- lines 12-14 make sense, but all other lines should be deleted. 
- lines 17 - 20 should be deleted because we do not want any inline styles! if need be, you can add this to your application.css
- as long as your asset pipeline is working, line 22 is already done for oyu, remove.